my_list.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'my_enum'
+
+class MyList
+  include MyEnumerable
+
+  def initialize(list)
+    @list = list
+  end
```

Comment:
Kindly, make sure to accept multiple arguments.
**Hint:** Consider doing something like this:
```
def initialize(*list)
```
That way it will accept multiple arguments.